mappings/net/minecraft/client/particle/ParticleMobApperance.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/bnj
+CLASS none/bnj net/minecraft/client/particle/ParticleMobApperance
```

Comment:
Though, would this be better as `ParticleMobAppear`, assuming it's used for when a mob appears? `Appearance` implies it has something to do with the way the mob looks.